app/assets/javascripts/sitewide/atwho_res.js.erb

```diff
@@ -162,8 +169,18 @@ var SmartAnnotation = (function() {
         })
 
         if ($currentAtWho.find('.tab-pane.active').length == 0) {
-          let filterType =  DEFAULT_SEARCH_FILTER;
-          $currentAtWho.find(`.${filterType.tag}`).click();
+          let filterType =  DEFAULT_SEARCH_FILTER.tag;
+          let teamId = $currentAtWho.find('.atwho-header-res').data('team-id');
+          let remeberedState = localStorage.getItem('scinote-smart-annotation-state-per-team-' + teamId);
+          if (remeberedState) {
+            try {
+              remeberedState = JSON.parse(remeberedState);
+              filterType = remeberedState.tag;
+              $currentAtWho.find(`.repository-object[data-object-id=${remeberedState.repository}]`)
+                .addClass('btn-primary');
+            } catch {}
```

Comment:
shouldn't it reset state if it is corrupted?

Reply:
filterType initially has a default state `let filterType =  DEFAULT_SEARCH_FILTER.tag;`